### PR TITLE
Enhancement: Adding Cloud CTA to OSS UI

### PR DESCRIPTION
**Pull request by WillRaphaelson**
For discussion: this change adds a small CTA to the sidebar and dashboard of the OSS UI

image

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [x] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: ui/src/pages/Dashboard.vue [hunk 1]
```diff
@@ -22,6 +22,13 @@
         </div>
       </template>
     </template>
+    <MarketingBanner title="Ready to scale?" subtitle="Automations, Serverless Work Pools, RBAC, and dedicated support on Prefect Cloud ">
+        <template #actions>
+          <p-button to="https://www.prefect.io/cloud" target="_blank" primary>
+            Upgrade to Cloud
+          </p-button>
+        </template>
+      </MarketingBanner>
```
Comment (znicholasbrown) [inline ui/src/pages/Dashboard.vue:31]:
```suggestion
    <MarketingBanner 
      title="Ready to scale?" 
      subtitle="Automations, Serverless Work Pools, RBAC, and dedicated support on Prefect Cloud"
    >
      <template #actions>
        <p-button to="https://www.prefect.io/cloud" target="_blank" primary>
          Upgrade to Cloud
        </p-button>
      </template>
    </MarketingBanner>
```

Just fixing spacing for the linter

File: ui/src/components/ContextSidebar.vue [hunk 2]
```diff
@@ -18,6 +18,17 @@
     <p-context-nav-item v-if="canSeeArtifacts" title="Artifacts" :to="routes.artifacts()" />
 
     <template #footer>
+      <a href="https://www.prefect.io/cloud" target="_blank" >
+        <p-context-nav-item >
```
Comment (znicholasbrown) [inline ui/src/components/ContextSidebar.vue:22]:
```suggestion
        <p-context-nav-item>
```

File: ui/src/components/ContextSidebar.vue [hunk 3]
```diff
@@ -18,6 +18,17 @@
     <p-context-nav-item v-if="canSeeArtifacts" title="Artifacts" :to="routes.artifacts()" />
 
     <template #footer>
+      <a href="https://www.prefect.io/cloud" target="_blank" >
```
Comment (znicholasbrown) [inline ui/src/components/ContextSidebar.vue:21]:
```suggestion
      <a href="https://www.prefect.io/cloud" target="_blank">
```

**Review by znicholasbrown — APPROVED** [review]

**Comment by joshy** [comment]
Is it possible to opt out?

**Comment by WillRaphaelson** [comment]
Hey @joshy, do you mean opt out of seeing the banner?

**Comment by WillRaphaelson** [comment]
We've not received other feedback on this being obtrusive, is this getting in the way of a piece of functionality for you? I could make it dismissible but by nature of it being self hosted we don't store or know any user presences so permanent opt out could get tricky.

**Comment by joshy** [comment]
The change is relatively new and I am not sure how often users are upgrading to the latest version. For a self-hosted version it should be possible to opt out. I can understand that you want to promote the cloud version but I am dealing with health data and the cloud is not an option in the foreseeable future. It would be nice to opt out but that is up to you. Thanks for considering it and your time dissusing it!

**Comment by knl** [comment]
I'd also like to have an option to opt out of seeing the banner -- we can't run anything off-prem, and this banner contains word "Upgrade", where stressed-out users may click because they think it'll upgrade their software.

**Pull request merged**